bpf/lib/common.h

```diff
@@ -539,8 +539,12 @@ enum {
 	CAPTURE_EGRESS = 2,
 };
 
+/* Minimum bytes for decoding packet headers during trace events.
+ * - Outer: 14 (ethernet) + 40 (ipv6) + 8 (udp) + 8 (geneve) + 24 (geneve v6 opts)
+ * - Inner: 14 (ethernet) + 40 (ipv6) + 8 (udp)
+ */
 #ifndef TRACE_PAYLOAD_LEN
-#define TRACE_PAYLOAD_LEN 128ULL
+#define TRACE_PAYLOAD_LEN 156ULL
```

Comment:
I wonder whether we really need to bump the default size. Or can have a `TRACE_PAYLOAD_OVERLAY_LEN`, and plug it in at the right places (based on `ctx_is_overlay()` etc).

Reply:
Thanks for the precious feedback. I tried to address it at my best, just updated the PR to achieve a similar behavior. 🙏🏼 

Closing this since outdated code, but super happy to start a new one!